### DISCUSSION TITLE
More Home Assistant Examples

### DISCUSSION THREAD

**Discussion by illuzn** [HASwitchPlate/openHASP#779] [Show and tell]
Are you looking for more sophisticated Home Assistant examples for the docs?

I made myself a pretty cool Alarm Control Panel which incorporates a screensaver with weather and entity status and a handy control panel for frequently use devices.

Happy to submit a PR.

**Comment by cbrochar** [comment]
I created a simple Alarm panel:
![alarmpanel](https://github.com/user-attachments/assets/fda544f8-ff54-4c4c-942e-0ff0c364cddc)

I had to create a few helpers and a script to validate the code entered to diarm/arm the Alarm panel. I would like to see other ways to do this. I took the ideas from the HASPOne blueprint on the Alarm Control panel.

I put the code here:

[https://github.com/cbrochar/openHasp_examples](url)
